site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -67,6 +67,22 @@ The form that will be used in the FusionAuth UI for adding and editing users.
 +
 include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
 
+===== Username settings
+
+Enable [field]#Unique usernames# to allow multiple users suggest the same username. If there are any collisions, FusionAuth will transparently create a unique username by appending a suffix. The user will continue to use the username without the suffix.
+
+[.api]
+[field]#Unique usernames# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.27.0#::
+When `true`, FusionAuth will handle username collisions by generating a random suffix.
```

Comment:
Does it make sense to include the additional information about how collisions work here (it is on the API already)?